### DISCUSSION TITLE
Email as a second factor in MFA

### DISCUSSION THREAD

**Discussion by abhishek-yelley-sails** [supabase/supabase#29082] [Feature Requests]
Supabase has an option to enable Multi-Factor Authentication (MFA) which is really useful. But it only supports TOTP (Authenticator Apps) and SMS (Phone and Whatsapp). It would be great if there is a way to receive a One-Time Password via email which can then be used to verify after a login method.

**Comment by mrtncode** [comment]
+1

**Comment by larakitb** [comment]
any updates on this feature request ?
